src/one_dragon/base/config/yaml_operator.py

```
@@ -94,16 +100,28 @@ def _get_write_path(self) -> str | None:
         if write_path is None:
             return
 
+        # 把要写入的内容转成字符串
+        new_content = yaml.dump(self.data, allow_unicode=True, sort_keys=False)
+        # 尝试读取旧文件内容
+        old_content = None
+        try:
+            with open(write_path, 'r', encoding='utf-8') as file:
+                old_content = file.read()
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)